src/Templating/Helper/PhoneNumberHelper.php

```diff
@@ -95,4 +102,25 @@ public function isType(PhoneNumber $phoneNumber, $type = PhoneNumberType::UNKNOW
 
         return $this->phoneNumberUtil->getNumberType($phoneNumber) === $type;
     }
+
+    /**
+     * @param PhoneNumber|string $phoneNumber
+     * @return PhoneNumber|void
+     * @throws \libphonenumber\NumberParseException
+     */
+    private function getPhoneNumber($phoneNumber)
+    {
+        if (is_string($phoneNumber)) {
+            $phoneNumber = $this->phoneNumberUtil->parse($phoneNumber);
+        }
+
+        if (!$phoneNumber instanceof PhoneNumber) {
+            throw new NumberParseException(
```

Comment:
I think this bundle should not throw an external exception, we should add an exception for this inside the bundle. Or throw directly an `InvalidArgumentException` (which already exists).

If the phoneNumber is a string (it is one most likely) printing it in the error message would be great.

Reply:
Printing error maybe unnoticed so I'd keep exception

Reply:
This is not my point. I'm saying the exception thrown here is not the right one and the message (inside the exception) may be better.

Reply:
I've changed it to `InvalidArgumentException` anyway.